### PR TITLE
[FSDE-11694] Add 4px border-radius to modal component

### PR DESCRIPTION
**Pull request by jllr**

### CODE REVIEW

**Review by janibolkvadze — APPROVED** [review]

File: src/chi/themes/lumen/_variables.scss [hunk 1]
```diff
@@ -972,6 +972,7 @@ $mobile-nav-list-item-active-box-shadow:    inset 0.125rem 0 0 $color-border-pri
 // Modal
 $modal-animated-transition-duration:        0.5s;
 $modal-background-color:                    $color-background-white;
+$modal-border-radius:                       0.25rem;
```
Comment (mattnickles) [inline src/chi/themes/lumen/_variables.scss:975]:
```suggestion
$modal-border-radius:                       $border-radius-base;
```

File: src/chi/themes/portal/_variables.scss [hunk 2]
```diff
@@ -972,6 +972,7 @@ $mobile-nav-list-item-active-box-shadow:    inset 0.125rem 0 0 $color-border-pri
 // Modal
 $modal-animated-transition-duration:        0.5s;
 $modal-background-color:                    $color-grey-10;
+$modal-border-radius:                       0.25rem;
```
Comment (mattnickles) [inline src/chi/themes/portal/_variables.scss:975]:
```suggestion
$modal-border-radius:                       $border-radius-base;
```

**Review by mattnickles — CHANGES_REQUESTED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-263/1/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://70aef87ea64f96d007200b1878dc5bc6c904018f-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-263-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Review by mattnickles — APPROVED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-263/2/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://e10bf4b97883f5f38879440bc823d3eb68f4e444-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-263-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Pull request merged**